src/App.js

```diff
@@ -1,10 +1,29 @@
 import 'antd/dist/antd.css';
-import React, { Suspense } from 'react';
-
+import React from 'react';
+import './App.css';
+import SingleActivityPage from './Pages/SingleActivityPage/SingleActivityPage';
 function App() {
+  const actPicArray = [
```

Comment:
you have a problem with the images URLs here, they are too long, also please move this data to the Mockdata folder